README.md

```diff
@@ -4,7 +4,7 @@
 
 [View the canonical, rendered version](https://danielkeep.github.io/tlborm/).
 
-This book is an attempt to distil the Rust community's collective knowledge of Rust macros.  As such, both additions (in the form of pull requests) and requests (in the form of issues) are welcome.
+This book is an attempt to distill the Rust community's collective knowledge of Rust macros.  As such, both additions (in the form of pull requests) and requests (in the form of issues) are welcome.
```

Comment:
It looks like this book is inactive unfortunately.

Note that the spelling of distil [is regional](https://www.collinsdictionary.com/dictionary/english/distil) in english, and I would guess that DanielKeep is using the correct form for his native english.

Reply:
colour me surprised!